packages/iov-ethereum/src/parse.ts

```diff
@@ -19,3 +21,43 @@ export class Parse {
     };
   }
 }
+
+export class Scraper {
+  public static parseBytesTx(bytes: PostableBytes, chainId: ChainId): SignedTransaction {
+    const json = JSON.parse(Encoding.fromUtf8(bytes));
+    const signature = new Uint8Array([]) as SignatureBytes;
```

Comment:
Don't we get the signature?

Reply:
Sadly not. A way to do this is get each full transaction from the node by calling searchTx by hash per each transaction, but also another way is to provide/get a scraper that returns signature values (r,s,v). We can define this step when we start to validate transactions into iov-core.

Reply:
Alright, thanks for feedback